OpenRA.Mods.Common/Traits/Render/RenderSprites.cs

```diff
@@ -36,7 +36,9 @@ public class RenderSpritesInfo : IRenderActorPreviewInfo, ITraitInfo
 		[PaletteReference] public readonly string Palette = null;
 
 		[Desc("Custom PlayerColorPalette: BaseName")]
-		public readonly string PlayerPalette = "player";
+		[PaletteReference("IsPlayerPalette")] public readonly string PlayerPalette = "player";
```

Comment:
Calling this `PlayerPalette` doesn't make sense after adding the `IsPlayerPalette` property.